chapters/en/chapter10/2.mdx

```diff
@@ -0,0 +1,63 @@
+# Set up your Argilla instance[[set-up-your-argilla-instance]]
+
+<CourseFloatingBanner chapter={10}
+  classNames="absolute z-10 right-0 top-0"
+  notebooks={[
+    {label: "Google Colab", value: "https://colab.research.google.com/github/huggingface/notebooks/blob/master/course/en/chapter10/section2.ipynb"},
+    {label: "Aws Studio", value: "https://studiolab.sagemaker.aws/import/github/huggingface/notebooks/blob/master/course/en/chapter10/section2.ipynb"},
+]} />
+
+To start using Argilla, you will need to set up your own Argilla instance first. Then you will need to install the Python SDK so that you can manage Argilla using Python code.
+
+## Deploy the Argilla UI
+
+The easiest way to set up your Argilla instance is through Hugging Face Spaces:
+
+1. Go to [Spaces](https://huggingface.co/spaces) in the Hugging Face Hub, and then click **Create new Space**.
```

Comment:
Maybe change to the direct link https://huggingface.co/new-space?template=argilla%2Fargilla-template-space?

Reply:
Ok, agreed. let's leave the explanation here but with a direct link to  https://huggingface.co/new-space?template=argilla%2Fargilla-template-space and not much explanation. And then I sentence to check the quickstart for further installation details. This way we can also remove the install with Docker part.